### PR TITLE
Add `team create` cli command

### PR DESCRIPTION
**Pull request by zapu**

### CODE REVIEW

File: protocol/avdl/keybase1/teams_ui.avdl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@namespace("keybase.1")
+
+protocol teams {
+  void teamCreate(int sessionID, string name);
+}
```
Comment (patrickxb) [inline protocol/avdl/keybase1/teams_ui.avdl]: Can you name this file `teams.avdl` instead of `teams_ui.avdl`?

Comment (zapu) [inline protocol/avdl/keybase1/teams_ui.avdl]: Sure, no problem. I think I got inspired by `chatUi` naming but then made it `protocol teams { ...` anyway in avd; file.

Comment (patrickxb) [inline protocol/avdl/keybase1/teams_ui.avdl]: Thanks.  `_ui.avdl` protocols are only for RPCs from service to clients.

Comment (zapu) [inline protocol/avdl/keybase1/teams_ui.avdl]: I see, thank you. Thanks for doing second look on this PR!

**Pull request merged**